README.md

```diff
@@ -16,7 +19,7 @@
 
 ## Loom Video 
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://www.loom.com/share/7a5ef9c89241452e809b352b34d858fc)
```

Comment:
- [ ] In the video please make sure you address the favuoute part of the project as stated in the project requirements.